resources/assets/lib/chat/chat-worker.ts

```diff
@@ -39,9 +39,7 @@ export default class ChatWorker implements DispatchListener {
 
     this.updateXHR = true;
 
-    const maxMessageId = this.channelStore.maxMessageId;
```

Comment:
apparently there's nothing else using `maxMessageId` anymore so it can be removed?